RequirementChecker.md

```diff
@@ -219,6 +219,26 @@ of conditions. Here are the current checks implemented.
 
   Checks that there are no references to `var` in the module.
 
+- `lineLength`
+
+  Example:
+
+      lineLength: 80
+  
+  Checks that there are no lines longer than 80 characters.
+
+-  `noWarnings`
+
+  Example:
+
+      noWarnings:
+        strict: false
+        exceptions: Defined but not used
+
+  If `strict` is true, checks that there are no warnings in the module.
+  If `strict` is false, checks that there are no warnings except for those 
+  matching a regular expression given by `exceptions`.
```

Comment:
Do you really need a boolean flag in addition to the exceptions field?  Wouldn't it be equivalent to just not specify any exceptions?

Why is `exceptions` a singular field?  It looks like a repeated field is better suited.